FY2013/2012-06.beancount

```diff
@@ -3,6 +3,38 @@
 option "title" "Gratipay Finances"
 option "operating_currency" "USD" ; The main currencies you use
 
+** Accounts ; Open all the accounts you have
```

Comment:
Since we want to use these accounts across all month files for the fiscal year, we should store them in a separate file. Under Ledger we were using [`FY2013.dat`](https://github.com/gratipay/finances/blob/dd633c305b4536069946dc35942bbc459b82aa3e/FY2013/FY2013.dat) and [`declarations.dat`](https://github.com/gratipay/finances/blob/dd633c305b4536069946dc35942bbc459b82aa3e/FY2013/declarations.dat). We will need to figure out a) what files it makes sense to use for Beancount, and b) how to make Beancount look at multiple files at once (simple `cat`?).